src/App/components/Card/index.js

```diff
@@ -13,30 +13,25 @@ const Card = ({ type, title, imgSrc, icon, highlightTitle, highlightSubTitle, te
         </div> : null}
         <div className="highlight">
             {icon ? <i className="material-icons">{icon}</i> : null}
-            <div className="highlight-wrapper">
-                {highlightTitle ? <h4 className="highlight-title">{highlightTitle}</h4> : null}
-                {highlightSubTitle ? <p className="highlight-subtitle">{highlightSubTitle}</p> : null}
-            </div>
+            {children}
```

Comment:
Any reason for `children` to be put in `.highlight` instead of `.card-body`?
Isn't the whole point of the card body to act as a container?